### PR TITLE
Script to consume and generate colors from figma API

### PR DESCRIPTION
**Pull request by f1lander**
### summary

In order to automate the design system process, getting the colors from Figma to our codebase is important

### how to test it

- Set in `FIGMA_ACCESS_TOKEN` in `.env`
- `npm run figma-colors`

###  pending

- group by color and rank
- generate a separate file called `colors.ts`
- spread those colors into stitches.config.theme.ts

### CODE REVIEW

File: src/ui/colors.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  /** Green  */
+  'Green/1': '#e5f7dd',
+  'Green/12': '#57b62b',
+  'Green/16': '#469222',
+  'Green/20': '#356e1a',
+  'Green/24': '#234911',
+  'Green/4': '#bae9a3',
+  'Green/8': '#74d447',
+  /** Grey  */
+  'Grey/1|Background': '#fbfbfb',
```
Comment (exrhizo) [inline src/ui/colors.ts:20]: Background is an alias of Grey, also would it be better to have:
`Grey1`
`grey-1`
`grey1`
I like the last

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "ethereumjs-util": "7.1.3",
     "ethers": "5.5.1",
     "faker": "5.5.3",
+    "figma-api": "1.9.2",
```
Comment (exrhizo) [inline package.json:36]: this could be moved to dev dependencies but it isn't a big deal

File: src/ui/scripts/figma-colors.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  fs.writeFileSync(path.resolve(`./src/ui/${fileName}`), fileContents);
+
+  // eslint-disable-next-line no-console
+  console.log(`Wrote ${content.length} colors to colors.ts`);
+};
+//#endregion Utils functions
+
+async function main() {
+  const api = new Figma.Api({
+    personalAccessToken: process.env.FIGMA_ACCESS_TOKEN || 'token',
```
Comment (exrhizo) [inline src/ui/scripts/figma-colors.ts:111]: Move to the top and give a short description of where to create this. Also add it to `.env.example`

Comment (exrhizo) [reply src/ui/scripts/figma-colors.ts:111]: Something like:
// Go to https://www.figma.com/developers/api and scrolled to find `personal access token`

**Review by exrhizo — APPROVED** [review]
Awesome, it worked on my setup :D

A few small changes requested

File: src/ui/colors.ts (hunk 1 above)
Comment (f1lander) [reply src/ui/colors.ts:20]: Sure, I'm missing some stuff that's why I still have it in WIP hehe, this was something that I wanted to ask you, so the idea will be to have `grey1` and `background` right?

File: package.json (hunk 2 above)
Comment (f1lander) [reply package.json:36]: yes, totally agree

File: src/ui/scripts/figma-colors.ts (hunk 3 above)
Comment (f1lander) [reply src/ui/scripts/figma-colors.ts:111]: Perfect.

File: src/ui/colors.ts (hunk 1 above)
Comment (exrhizo) [reply src/ui/colors.ts:20]: yes

**Review by levity — APPROVED** [review]

**Pull request merged**